crates/sui-protocol-config/src/lib.rs

```diff
@@ -146,6 +146,11 @@ pub struct ProtocolConfig {
     /// Maximum number of individual transactions in a Batch transaction.
     max_tx_in_batch: Option<u32>,
 
+    /// Maximum number of mutable input objects.
+    max_mutable_inputs: Option<u64>,
+
+    /// Maximum size of serialized transaction effects.
+    max_serialized_tx_effects_size: Option<usize>,
```

Comment:
can you add _bytes suffix since I have https://github.com/MystenLabs/sui/pull/9025 coming for the others?